evap/staff/tests/test_views.py

```diff
@@ -1923,6 +1923,99 @@ def test_evaluation_copy(self):
         self.assertEqual(copied_evaluation.contributions.count(), 4)
 
 
+class TestEvaluationExamCreation(WebTestStaffMode):
+    csrf_checks = False
+    url = reverse("staff:create_exam_evaluation")
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.course = baker.make(Course)
+        vote_start_datetime = datetime.datetime.now() - datetime.timedelta(days=50)
+        cls.evaluation = baker.make(Evaluation, course=cls.course, vote_start_datetime=vote_start_datetime)
+        cls.contributions = baker.make(
+            Contribution, evaluation=cls.evaluation, _fill_optional=["contributor"], _quantity=3, _bulk_create=True
+        )
+        cls.exam_datetime = datetime.date.today() + datetime.timedelta(days=10)
+        cls.semester_overview_url = reverse("staff:semester_view", args=[cls.course.semester.pk])
+        # By default the evaluation does not have any participants, so we need to add some
+        cls.evaluation.participants.set(baker.make(UserProfile, _quantity=3))
+        cls.params = {"evaluation_id": cls.evaluation.pk, "exam_date": cls.exam_datetime}
+
+    def test_create_exam_evaluation(self):
+        self.app.post(
+            self.url,
+            user=self.manager,
+            status=200,
+            params=self.params,
+        )
+        self.assertEqual(Evaluation.objects.count(), 2)
+        exam_evaluation = Evaluation.objects.exclude(pk=self.evaluation.pk).get()
+        self.assertEqual(exam_evaluation.contributions.count(), self.evaluation.contributions.count())
+        self.assertEqual(
+            exam_evaluation.vote_start_datetime,
+            datetime.datetime.combine(self.exam_datetime + datetime.timedelta(days=1), datetime.time(8, 0)),
+        )
+        self.assertEqual(exam_evaluation.vote_end_date, self.exam_datetime + datetime.timedelta(days=3))
+        self.assertEqual(exam_evaluation.name_de, "Klausur")
+        self.assertEqual(exam_evaluation.name_en, "Exam")
+        self.assertEqual(exam_evaluation.course, self.evaluation.course)
+        self.assertQuerySetEqual(exam_evaluation.participants.all(), self.evaluation.participants.all())
+        self.assertEqual(exam_evaluation.weight, 1)
+
+        evaluation = Evaluation.objects.get(pk=self.evaluation.pk)
+        self.assertEqual(evaluation.weight, 9)
+
+    def test_exam_evaluation_for_single_result(self):
+        self.evaluation.is_single_result = True
+        self.evaluation.save()
+        self.app.get("", user=self.manager)  # Needed to not get a last login database update
+        with assert_no_database_modifications():
+            self.app.post(
+                self.url,
+                user=self.manager,
+                status=400,
+                params=self.params,
+            )
```

Comment:
```suggestion
            self.app.post(self.url, user=self.manager, status=400, params=self.params)
```
(the trailing comma makes black format it in one-line-per-argument fashion)
(repeated below)